README.md

```diff
@@ -18,19 +18,232 @@ Alternatively, you can use the Open in Colab browser extension ([Chrome](https:/
+| section          | text | The first part of the JSON path to the field in the data, e.g. budget.                                                                                                                                                                                  |
+| path             | text | The JSON path to the field in the data, e.g. budget/description.                                                                                                                                                                                        |
+| title            | text | The field’s title in the JSON schema. If the field has no title, defaults to the field’s name followed by “*”.                                                                                                                                          |
+| description      | text | The field’s description in the JSON schema. URLs are removed (see the links column).                                                                                                                                                                    |
+| type             | text | A comma-separated list of the field’s type in the JSON schema, excluding “null”. If the field has no type, defaults to “unknown”.                                                                                                                       |
+| range            | text | The field’s allowed number of occurrences: “0..1” if the field defines an optional literal value, “0..n” if the field defines an optional array, “1..1” if the field defines a required literal value, or “1..n” if the field defines a required array. |
+| values           | text | If the field’s schema sets: format - the format; pattern - the pattern; enum - “Enum: “ followed by the enum as a comma-separated list, excluding null; items/enum - “Enum: “ followed by the items/enum as a comma-separated list, excluding null.     |
+| links            | text | The URLs extracted from the field’s description.                                                                                                                                                                                                        |
+| deprecated       | text | The OC4IDS minor version in which the field (or its parent) was deprecated.                                                                                                                                                                             |
+| deprecationnotes | text | The explanation for the deprecation of the field.                                                                                                                                                                                                       |
```

Comment:
```suggestion
| deprecation_notes | text | The explanation for the deprecation of the field.                                                                                                                                                                                                       |
```

Not sure if this is right but based on the other column names I think this is needed

Reply:
For this table, the column names are taken directly from the output of OCDS Kit's `mapping-sheet` command so I think it's better to leave it as is.

Reply:
ah, okay sounds good